content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+        * Authentication: Authenticate through HTTP Basic Authentication with a username and a password.
+        * Cookies: Defined cookies are added to the default browser cookies. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+
+2. **Name**: Set the name of your browser test.
+3. **Select your tags**: Select the tags attached to your browser test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the Synthetics page.
+4. **Devices**: Select which devices to run your check on. Available devices are `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][5]. You can also set up a [Private Location][11] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every 15 minutes to once per week. [Contact support][12] to enable additional frequencies for your test.
+7. **Alert conditions**: Set the conditions that need to be met (`minutes`, `locations`, and `retry`) for a test to retry if it has failed.
+8. **Notify your team**: Enter an message associated with the alert condition and who the alert should be sent to. Use `@` to tag people in the body of the message and select, from the dropdown list, who to send the alert to. You can also type in an email address in this location to send to any team or member that is not listed.
+9. Once you've input all required fields, click the *Save Details & Record Test* button.
+
+#### Record a test with variables and assertions
+
+Tests can be only recorded from **[Google Chrome][1]**. To record your test, download the [Datadog Record Test extension for Google Chrome][2].
+
+{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your browser test.
+3. Your actions are recorded and used to create steps within your browser test scenario. You can record the uploading of files as an action, though this is limited to 10 files, with a limit of 5MB each.
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_tests/browser_test_step.png" alt="Browser Test steps" responsive="true" style="width:80%;">}}
+**Note**: **Your last browser test step must be an Assertion**, otherwise there is nothing to check.
```

Comment:
**Note**: **Your last browser test step must be an Assertion**, this will ensure your test ended up on the expected page and found the expected element. This is what will allow your test to be self-maintaining as it will be able to use a different set of keys to perform the recorded journey and confirm it landed on the expected element although some changes on the UI might have happened since the test was first recorded.